card/mod/machines/lib/javascript/decko/editor.js.coffee

```diff
@@ -30,35 +35,21 @@ jQuery.fn.extend {
     new_val = fn.call this
     field.val new_val
     field.change() if init_val != new_val
-}
-
-doubleClickActiveMap = {off: false, on: true, signed_in: decko.currentUserId}
 
-doubleClickActive = () ->
-  doubleClickActiveMap[decko.doubleClick]
-  # else alert "illegal configuration: " + decko.doubleClick
+$(window).ready ->
+  decko.initializeEditors $('body')
+  # setTimeout (-> decko.initializeEditors $('body')), 10
+  # dislike the timeout, but without this forms with multiple TinyMCE editors
+  # were failing to load properly
+  # I couldn't reproduce that problem described above -pk
```

Comment:
! (first thing to check if problems with tinymce come up)

Reply:
✅